interlay/interbtc-ui
# fix(Loans): rewards apy

# Interbtc UI Pull Request Template

## Description

Function used to calculate rewards apy was not being used correctly.

## Current behaviour (updates)

Wrong parameters that result in wrong APY

## New behaviour

Uses correct parents

## Reproducible testing steps:

- Hover Loans Tables APY 
- Open table row and check the Loan Modal
